CONTRIBUTING.md

```diff
@@ -31,8 +106,13 @@ Please contact us if you need help or have ideas to improve this guide.
 * Check for **lint** errors.
 * Create and run **tests**.
 
+## How to use Git
```

Comment:
Personally I wouldn't put that much documentation about Git here and simply refer to existing tutorials regarding feature branches, forks, etc. However, you're reviewing most of the contributions, so if you think that this will help you then feel free to keep it in.